apps/services/sessions/src/app/sessions/sessions.controller.spec.ts

```diff
@@ -1,34 +1,46 @@
 import { getQueueToken } from '@nestjs/bull'
+import { AuthDelegationType } from 'delegation'
```

Comment:
I was surprised that this worked, but saw the reason in the first `paths` definition in `tsconfig.base.json`. But I would rather suggest to use the `@island.is` import path, as this can be confused as some npm package.
```suggestion
import { AuthDelegationType } from '@island.is/shared/types'
```

Reply:
Fixed.